spec/models/tasks/send_cavc_remand_processed_letter_task_spec.rb

```diff
@@ -121,4 +123,52 @@
       end
     end
   end
+
+  describe "#available_actions_unwrapper" do
+    let(:cavc_task) { create(:send_cavc_remand_processed_letter_task, assigned_to: create(:user)) }
+
+    subject { cavc_task.available_actions_unwrapper(cavc_task.assigned_to) }
+
+    before do
+      # Create completed distribution task to make sure we're picking the correct (open) parent
+      completed_distribution_task = build(:task, appeal: cavc_task.appeal, type: DistributionTask.name)
+      completed_distribution_task.save!(validate: false)
+      completed_distribution_task.completed!
+    end
+
+    it "provides the parent id as the open distribution task for blocking admin actions" do
```

Comment:
optional wording suggestion:
```suggestion
    it "uses the open distribution task as the parent for blocking admin action tasks" do
```